docs/installation/expansion.rst

```diff
@@ -65,12 +49,23 @@ The MetalK8s GUI provides a simple form for that purpose.
    - **SSH Key Path**: the path to the private key generated in
      :ref:`this procedure <Bootstrap SSH Provisioning>`
    - **Sudo required**: whether the SSH deployment will need ``sudo`` access
-   - **Roles/Workload Plane**: check this box if the new Node should receive
-     workload applications
-   - **Roles/Control Plane**: check this box if the new Node should run
-     control-plane services
-   - **Roles/Infra**: check this box if the new Node should run
-     infra services
+   - **Roles/Workload Plane**: if you select this box, you will remove all the
+     taints and enable the node role if Infra Box isn't selected.
+   - **Roles/Control Plane**: if you select this box, you enable both role
+     and taint for master and etcd.
+   - **Roles/Infra**: if you select this box, you enable both role and taint
+     for infra. ( If Control Plane box is also selected, you will remove the
+     taints for master and etcd).
+
+  There are three examples to help you determine what is installed and deployed
+  on the Node.
+
+   * Select **Control Plane** checkbox: master and etcd services run on this
+     Node.
+
+   * Select **Infra** checkbox: infra services run on this Node.
+   * Select **Work Plane** and **Infra** checkbox: infra services and workload
```

Comment:
I would just keep one example as a Note

Reply:
IMHO, I still would like to keep the three examples for IT Generalist. It also corresponds to the three examples in CLI. What do you think?

Reply:
ok. So I leave one example to show the combination of different roles is enable.